src/gradualizer_prelude.erl

```diff
@@ -2,8 +2,70 @@
 
 %% This module contains specs to replace incorrect or inexact specs in OTP.
 
+-type ext_binary() :: binary().
+
+-spec erlang:apply(function(), [any()]) -> any().
+-spec erlang:apply(atom(), function(), [any()]) -> any().
```

Comment:
`function()` means a fun object, I think this should be `atom()` (but could use `module()` as first arg as that is an alias for `atom()`)

Reply:
Indeed. Thanks!